packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: modified `plugins validate` to default to fail when an error occurs. `zowe plugins validate <plugin-name>` behaves as `zowe plugins validate <plugin-name> --fail-on-error` [#2221](https://github.com/zowe/zowe-cli/pull/2221)
```

Comment:
Can we say what `plugins validate` is?: Modified the `plugins validate` **command/environment variable/pasta dish** to default to fail ... .
Also: The `zowe plugins validate <plugin-name>` **command** now behaves as the **command** `zowe plugins validate <plugin-name> --fail-on-error`.